src/documents/management/commands/document_thumbnails.py

```diff
@@ -21,6 +22,13 @@ def _process_document(doc_in):
         return
 
     try:
+
+        existing_thumbnail = Path(document.thumbnail_path).resolve()
+
+        # Remove an existing PNG format thumbnail, if it existed
+        if existing_thumbnail.exists() and existing_thumbnail.suffix == ".png":
```

Comment:
this code is a bit surprising here - should it go to `get_optimised_thumbnail` instead?